userguide/conf.py

```diff
@@ -95,6 +95,7 @@
 .. |help-pin|             replace::  (Pin)
 .. |help-text|            replace::  (Help Text)
 .. |help-close|           replace::  (Close)
+.. |human-readable|       replace:: This is a :ref:`human readable field <Human Readable Fields>`.
```

Comment:
Sorry I didn't think of this earlier, but the correct term for these is "humanized".
```suggestion
.. |humanized-field|       replace:: This is a :ref:`human readable field <Humanized Fields>`.
```